### ISSUE TITLE
Regression: DocCommentSpacing treats phpcs:* comments differently since since 8.13.0

### ISSUE THREAD

**Issue by TravisCarden** [slevomat/coding-standard#1594]
The doc comment parsing in https://github.com/slevomat/coding-standard/releases/tag/8.13.0 seems to have introduced a regression whereby `phpcs:*` comments can incorrectly trigger `SlevomatCodingStandard.Commenting.DocCommentSpacing.IncorrectLinesCountAfterLastContent`. Here's a test case:

```php
/**
 * This is a class.
 *
 * phpcs:disable SlevomatCodingStandard.Classes.RequireAbstractOrFinal.ClassNeitherAbstractNorFinal
 */
class Test implements HostInterface
{
    /**
     * This is a function.
     *
     * phpcs:disable SlevomatCodingStandard.Functions.DisallowEmptyFunction.EmptyFunction
     */
    public function test(): void {}
}
```

As of `8.13.0`, this produces the following errors:

```
FILE: Example.php
-----------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 4 LINES
-----------------------------------------------------------------------------------------------------------------------------------------------------------
  9 | ERROR | [x] Expected 0 lines after last content, found 2.
    |       |     (SlevomatCodingStandard.Commenting.DocCommentSpacing.IncorrectLinesCountAfterLastContent)
 16 | ERROR | [x] Expected 0 lines after last content, found 2.
    |       |     (SlevomatCodingStandard.Commenting.DocCommentSpacing.IncorrectLinesCountAfterLastContent)
-----------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------------------------------------------------------------------------------
```

If I run PHPCBF it removes the "offending" comments:

```diff
 /**
  * This is a class.
- *
- * phpcs:disable SlevomatCodingStandard.Classes.RequireAbstractOrFinal.ClassNeitherAbstractNorFinal
  */
 class Test implements HostInterface
 {
     /**
      * This is a function.
-     *
-     * phpcs:disable SlevomatCodingStandard.Functions.DisallowEmptyFunction.EmptyFunction
      */
     public function test(): void {}
 }
```

If I either remove the `phpcs:*` bits or prefix them with an "at" sign (`@`), they aren't treated as errors:

```diff
 /**
  * This is a class.
  *
- * phpcs:disable SlevomatCodingStandard.Classes.RequireAbstractOrFinal.ClassNeitherAbstractNorFinal
+ * @phpcs:disable SlevomatCodingStandard.Classes.RequireAbstractOrFinal.ClassNeitherAbstractNorFinal
  */
 class Test implements HostInterface
 {
     /**
      * This is a function.
      *
-     * phpcs:disable SlevomatCodingStandard.Functions.DisallowEmptyFunction.EmptyFunction
+     * SlevomatCodingStandard.Functions.DisallowEmptyFunction.EmptyFunction
      */
     public function test(): void {}
 }
```

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/pull/1599
